src/main/webapp/views/scheduling.jsp

```diff
@@ -0,0 +1,152 @@
+<%@ page language="java" contentType="text/html; charset=ISO-8859-1" pageEncoding="ISO-8859-1" isELIgnored="false"%>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core"  %>
+<html>
```

Comment:
508 error missing default lang attribute. 
I'm seeing a lot of missing meta here.  Run your whole loaded browser source through https://validator.w3.org 
and use Axe Devtools to perform a full 508 accessibility compliance check on your webpage.  
https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US 
Fix all errors.